packages/terra-form-field/src/Field.scss

```diff
@@ -24,7 +24,7 @@
   }
 
   .label {
-    color: var(--terra-form-field-label-color, rgba(0, 0, 0, 0.8));
```

Comment:
Should the theme changes in this file be part of this work?

Reply:
Talked to @neilpfeiffer about accessibility issues with help text and error text colors and based on suggestions from UX, had to change the color of label too.